xclim/indicators/atmos/_temperature.py

```diff
@@ -215,6 +216,21 @@ class TempWithIndexing(ResamplingIndicatorWithIndexing):
     compute=indices.heat_wave_frequency,
 )
 
+heat_wave_magnitude = Temp(
+    title="Heat wave magnitude",
+    identifier="heat_wave_magnitude",
+    units="K d",
+    long_name="Maximum difference between daily maximum temperature and {thresh} for days within a heat wave. "
+    "A heat wave is defined as a series of at least {window} consecutive days with daily maximum temperature above "
+    "{thresh}.",
+    description="Magnitude of the most intensive heat wave per {freq} as accumulated deg C above a given threshold "
```

Comment:
`deg C` in description, but units above are specified as  `K d`.

Reply:
Fixed, thx.